script/QEMU.sh

```diff
@@ -66,9 +73,15 @@ QEMU_ARGS+=" -drive file=$DRIVE,index=0,media=disk,format=raw"
 QEMU_ARGS+=" -serial $SERIAL"
 QEMU_ARGS+=" -device isa-debug-exit,iobase=0xf4,iosize=0x04"
 # Expose standard gdb on :1234
-QEMU_ARGS+=" -s"
+if [ "$GDB" -gt 0 ]; then
+  QEMU_ARGS+=" -s"
+fi
 # Pause on startup (require 'c' on console, or 'c' in GDB to start)
 if [ "$PAUSE" -gt 0 ]; then
+  if [ "$GDB" -eq 0 ]; then
+    echo "Error: -S requires a gdb stub to do anything."
```

Comment:
technically this isn't true, but it's not actually *useful* if you don't have the gdb stub.

you can press 'c' in the qemu console to continue execution.